R/test-that.R

```diff
@@ -167,6 +167,11 @@ test_code <- function(code, env, reporter = NULL, skip_on_empty = TRUE) {
           }
         },
         expectation = handle_expectation,
+        packageNotFoundError = function(e) {
+          if (on_cran()) {
+            skip(paste0(e$package, " is not installed."))
```

Comment:
Maybe sync it exactly with the `skip_if_not_installed()` message?

https://github.com/r-lib/testthat/blob/60998023ddb0a1fd812bc66b26e47cb66f219498/R/skip.R#L107